pontificia-universidade-catolica-do-parana-abnt.csl

```diff
@@ -1,216 +1,465 @@
-    <names variable="author">
-      <name form="short" name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="never">
-        <name-part name="family" text-case="capitalize-first"/>
-        <name-part name="given" text-case="capitalize-first"/>
-      </name>
+  <macro name="recipient">
+    <!-- Macro responsavel por mostrar o nome do destinatário -->
+    <text term="recipient" suffix=": "/>
+    <names variable="recipient" delimiter="; ">
+      <name and="text" sort-separator=" " delimiter-precedes-last="always"/>
       <et-al font-style="italic"/>
-      <substitute>
-        <names variable="editor"/>
-        <choose>
-          <if type="book">
-            <text variable="title" form="short"/>
-          </if>
-          <else>
-            <text variable="title" form="short" text-case="uppercase" quotes="false"/>
-          </else>
-        </choose>
-      </substitute>
     </names>
   </macro>
-  <macro name="director">
-    <names variable="director">
-      <label form="verb" text-case="capitalize-first" suffix=": "/>
-      <name sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
-        <name-part name="given" text-case="capitalize-first"/>
-        <name-part name="family" text-case="capitalize-first"/>
-      </name>
+  <macro name="translator">
+    <!-- Macro responsavel por mostrar o nome do tradutor -->
+    <text term="translator" suffix=": "/>
+    <names variable="translator" delimiter=", ">
+      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"/>
+      <et-al font-style="italic"/>
     </names>
   </macro>
+  <macro name="author-short">
+    <!-- Macro responsável por mostrar o autor na citação -->
+    <choose>
+      <if type="broadcast motion_picture song" match="any">
+        <text variable="title" form="short" text-case="uppercase" quotes="false"/>
+      </if>
+      <else-if type="musical_score" match="any">
+        <names variable="composer">
+          <name form="short" delimiter="; " delimiter-precedes-last="never" initialize-with=". " name-as-sort-order="all">
+            <name-part name="family"/>
+            <name-part name="given"/>
+          </name>
+        </names>
+      </else-if>
+      <else>
+        <names variable="author">
+          <name form="short" delimiter="; " delimiter-precedes-last="never" initialize-with=". " name-as-sort-order="all">
+            <name-part name="family"/>
+            <name-part name="given"/>
+          </name>
+          <et-al font-style="italic"/>
+          <substitute>
+            <names variable="editor"/>
+            <names variable="translator"/>
+            <choose>
+              <if type="book">
+                <!-- uso do título quando não houver autor -->
+                <text variable="title" form="short"/>
+              </if>
+              <else>
+                <text variable="title" form="short" quotes="false"/>
```

Comment:
this conditional doesn't do anything since both the if and the else have the same output. Why was this introduced?